sdk/identity/tests.yml

```diff
@@ -31,6 +31,6 @@ extends:
           # Contains alternate tenant, AAD app and cert info for testing
           - $(sub-config-identity-test-resources)
         ServiceConnection: azure-sdk-tests
-    EnvVars:
```

Comment:
Don't we still want the access token, or is that plumbed through underneath now?

Reply:
It's plumbed through underneath now: https://github.com/Azure/azure-sdk-for-net/blob/main/eng/pipelines/templates/jobs/live.tests.yml#L207 

This gives the following error: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3911090&view=results